src/services/UserCreator.php

```diff
@@ -54,7 +54,7 @@ public function create(): int
             (new UserParams('lastname', $this->reqBody['lastname']))->getContent(),
             // password is never set by admin/sysadmin
             '',
-            Check::usergroup($this->requester, Usergroup::from((int) ($this->reqBody['usergroup'] ?? 4)))->value,
+            Check::usergroup($this->requester, Usergroup::from((int) ($this->reqBody['usergroup'] ?? 4))),
```

Comment:
The 4 here, that's a `Usergroup::User`, right?

Reply:
Yes, you are right. We identified the confusion arrising by using integers. This is addressed in #4993.

Reply:
I agree that it should be addressed in #4993. Nice initiative, by the way!